### DISCUSSION TITLE
How to specify github branch? Production Branch option not appearing.

### DISCUSSION THREAD

**Discussion by BlankAdventure** [vercel/community#6568] [Help] [answered]
### Summary

I can't for the life of me find a way to specify a particular branch. Yes, I have see this link:

https://vercel.com/guides/can-i-use-a-non-default-branch-for-production#production-branch-project-setting

However I can't find that page/option anywhere! I'm on the hobby plan, so maybe that has something to do with it?

### Steps to Reproduce

None

**Comment by amyegan** [comment, marked as answer]
Hey @BlankAdventure. You can set a custom production branch on any plan. I can walk you through it. Please don't be offended by my ELI5 approach. I just want to make sure I don't miss sharing the particular info you need. 🙂 

1. Start from your dashboard. It should be what you see when you are logged in and visit [vercel.com](https://vercel.com)
2. Click the project name to see its details. The URL would be like `vercel.com/{TEAM-NAME}/{PROJECT-NAME}`
3. Go to the **Settings** tab. It's the last option in the series of tabs near the top of the page.
4. Open **Git** settings from the navigation list on the lefthand side of the page.
5. Scroll down until you see the **Production Branch** box. If you have a git repo connected, that's where you can change which branch is used for production.
<img width="1216" alt="" src="https://github.com/vercel/community/assets/8247105/05ae37ec-351e-4c60-8abd-d0d3832e8ba1">

**Comment by BlankAdventure** [comment]
Thanks, I was able to find it.
